incubator/README.md

```diff
@@ -239,9 +167,18 @@ get full support by PingCAP and the community.
   - The participant fully as an organizer or community member
   - The project will be maintained continuously after it is finished
 
-## Features and Projects
+The TiDB community will provide marketing supports for successfully incubated projects to help them quickly gain the attention of the community and the public. Marketing supports mainly include but not limited to: emails, blogs, online media, online and offline events.
+
+- Marketing supports for graduated projects
+  - Announce the graduation of a project to the community through newsletters, Slack, mailing lists, WeChat groups, etc.
+  - Announce the graduation of the project through PingCAP's official WeChat account, blog, Weibo, Zhihu, Twitter, and other self-publishing channels.
```

Comment:
I think not only pingcap social media channel, but also the tidb community media resourses

Reply:
Fixed in https://github.com/pingcap/community/pull/375/commits/23409d2b9a77f0e5272f4fd9ef02d77d8333f071